src/components/Header/styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const StyledHeader = styled.header`
+	display: flex;
+	flex-direction: row;
```

Comment:
You don't necessarily need to add the `flex-direction: row;` as it is the default direction of `display: flex;`.